reactor-netty-http/src/test/java/reactor/netty/http/server/HttpServerTests.java

```diff
@@ -2613,4 +2614,46 @@ private void doTestConnectionClosePropagatedAsError(String request) throws Excep
 				.isInstanceOf(AbortedException.class)
 				.hasMessage("Connection has been closed");
 	}
+
+	@Test
+	void testRemoveRoutes() {
```

Comment:
Please add a suppression for the warning below as this is expected

```
/.../reactor-netty/reactor-netty-http/src/test/java/reactor/netty/http/server/HttpServerTests.java:2643: warning: [ReferenceEquality] Comparison using reference equality instead of value equality
                                && metadata.getMethod() == HttpMethod.GET);
                                                        ^
    (see https://errorprone.info/bugpattern/ReferenceEquality)
  Did you mean '&& Objects.equals(metadata.getMethod(), HttpMethod.GET));' or '&& metadata.getMethod().equals(HttpMethod.GET));'?
/.../reactor-netty/reactor-netty-http/src/test/java/reactor/netty/http/server/HttpServerTests.java:2650: warning: [ReferenceEquality] Comparison using reference equality instead of value equality
                                .expectNextMatches(response -> response.status() == HttpResponseStatus.NOT_FOUND)
                                                                                 ^
    (see https://errorprone.info/bugpattern/ReferenceEquality)
  Did you mean '.expectNextMatches(response -> Objects.equals(response.status(), HttpResponseStatus.NOT_FOUND))' or '.expectNextMatches(response -> response.status().equals(HttpResponseStatus.NOT_FOUND))'?
2 warnings
```